packages/oc-pages/project_overview/components/template/template_buttons.vue

```diff
@@ -125,21 +149,30 @@ export default {
             >
 
         </div>
-        <div class="d-flex">
-            <gl-button
-                v-show="deployStatus != 'hidden' && !editingTorndown"
-                title="Deploy"
-                :aria-label="__('Deploy')"
-                data-testid="deploy-button"
-                type="button"
-                icon="upload"
-                class="deploy-action"
-                :disabled="deployStatus == 'disabled'"
-                @click.prevent="triggerDeploy"
-                >{{ __('Deploy') }}</gl-button
-            >
-            <!--gl-button v-else type="button" class="deploy-action" loading>{{ __('Deploying...') }}</gl-button-->
+        <el-tooltip :disabled="!deployTooltip">
+            <template #content>
+                <div>
+                    {{deployTooltip}}
+                </div>
+            </template>
+            <div class="d-flex flex-column position-relative">
+                <gl-button
+                    v-show="deployStatus != 'hidden' && !editingTorndown"
+                    :aria-label="__('Deploy')"
+                    data-testid="deploy-button"
+                    :title="!deployTooltip? 'Deploy': null"
+                    type="button"
+                    icon="upload"
+                    class="deploy-action"
+                    :disabled="deployStatus == 'disabled'"
+                    @click.prevent="triggerDeploy"
+                >
+                    {{ __('Deploy') }}
+                </gl-button>
+                <error-small class="position-absolute" style="top: 2em; right: 0; width: 300px; text-align: right;" :condition="!canDeploy">Deployment is invalid</error-small>
```

Comment:
how about this instead?

 `<error-small class="position-absolute" style="top: 2em; right: 0; width: 300px; text-align: right;" :condition="!canDeploy">Deployment has errors.<i class="el-icon-info"></i></error-small>`

Reply:
I can narrow the criteria for that, but it doesn't always have errors in this state.

Reply:
I just changed the message text and add the info icon to make the tooltip obvious... At least that's all I intended to do